### PR TITLE
Remove table numbering if ant parameter 'table-numbering' equal 'no' …

### PR DESCRIPTION
**Pull request by yuliia-biletska**
…or 'false'

### CODE REVIEW

**Review by vasyl-krupa — APPROVED** [review]
looks good

File: plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/cover.xsl"/>
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_common.xsl"/>
+  <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_tables.xsl"/>
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl]: DITA-OT has default filename for tables. We don't need 'dita2xmlrocks_' prefix here

File: plugins/rocks.xml.webhelp/xsl/dita2xmlrocks_tables.xsl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                xmlns:related-links="http://dita-ot.sourceforge.net/ns/200709/related-links"
+                exclude-result-prefixes="#all"
+                version="2.0">
+
+    <xsl:param name="table-numbering"/>
+
+
+    <xsl:template match="*[contains(@class, ' topic/table ')]/*[contains(@class, ' topic/title ')]" mode="title-number">
+        <xsl:param name="number" as="xs:integer"/>
+        <xsl:choose>
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/dita2xmlrocks_tables.xsl]: @Yuliia-Bondarenko-1806 please exclude all 'Table #.' string (or inline element with this string if exists)?

File: plugins/rocks.xml.webhelp/xsl/dita2xmlrocks_tables.xsl [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                xmlns:related-links="http://dita-ot.sourceforge.net/ns/200709/related-links"
+                exclude-result-prefixes="#all"
+                version="2.0">
+
+    <xsl:param name="table-numbering"/>
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/dita2xmlrocks_tables.xsl]: We don't need this param here if we add the same in other file

**Comment by yuliia-biletska** [comment]
done

File: plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/cover.xsl"/>
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_common.xsl"/>
-  <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_tables.xsl"/>
+  <xsl:import href="plugin:rocks.xml.webhelp:xsl/tables.xsl"/>
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl]: I don't think we need tables transformation imported from the cover xsl.

Comment (yuliia-biletska) [reply plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl]: done

**Review by ibarinova — APPROVED** [review]
Approved

**Pull request merged**